pages/create-account.tsx

```diff
@@ -56,38 +73,50 @@ export default function CreateAccount() {
       console.log(`${field}: ${input}`);
     });
     // Update the error messages, telling React to re-render the component
-    setErrorMessages(errors);
+    setErrorMessages({"confirmPassword" : errors["confirmPassword"],
+                      "email" : errors["email"],
+                      "firstName" : errors["firstName"],
+                      "lastName" : errors["lastName"],
+                      "password" : errors["password"],
+                      "username" : errors["username"]
+                      });
 
     // Check if the form has any errors
     const hasNoErrors = Object.values(errors)
                                 .every((input) => input === '');
+
+    const username = formData.get("username").toString()
+    const password = formData.get("password").toString()
+    const email = formData.get("email")
+    const firstName = formData.get("firstName")
+    const lastName = formData.get("lastName")
```

Comment:
Any reason only `username` and `password` need the `toString()` method?

Reply:
formData.get() returns a type of formDataEntryValue. Typescript now enforces the type for the Auth methods to string. Doesn't seem to care about the attributes, but does care about username and password. I'll just make them all strings for consistency.